chromium_src/chrome/browser/unexpire_flags.cc

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/common/channel_info.h"
+#include "components/version_info/version_info.h"
+
+#define IsFlagExpired IsFlagExpired_ChromiumImpl
+#include "../../../../chrome/browser/unexpire_flags.cc"
+#undef IsFlagExpired
+
+namespace flags {
+
+bool IsFlagExpired(const flags_ui::FlagsStorage* storage,
```

Comment:
omg, maybe we can just write a separate function like `IsPlaylistEnabled()` that would check the feature and the channels? Without overriding.

Reply:
To make `playlist` feature visible in brave://flags for nightly channel, this is introduced.